packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterInputMultipleValueProps.ts

```diff
@@ -0,0 +1,8 @@
+import { GridFilterItem } from '../../../models/gridFilterItem';
+
+export interface GridFilterInputMultipleValueProps {
```

Comment:
The props should be inside `GridFilterInputMultipleValue`. `GridFilterInputValue` is not a good example.